packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -1797,4 +1837,40 @@ describe.each([
           { two: "foo", other: [{ _id: otherRows[0]._id }] },
         ]))
     })
+
+  // lucene can't count, and in memory there is no point
```

Comment:
I would still test this for in-memory.

Reply:
It doesn't have any implementation for counting, do you think we should add it?